keras_cv/models/__internal__/unet.py

```diff
@@ -177,6 +177,10 @@ def UNet(
     All function parameters require curried functions as inputs which return a
     function that acts on tensors as inputs.
 
+    Reference:
+        - [U-Net: Convolutional Networks for Biomedical Image Segmentation]
+          (https://arxiv.org/abs/1505.04597)
```

Comment:
We'll need to keep these in the same line.

Reply:
With The Original Link, keeping in the same line would be difficult. Since its give a Linter Issue. 
An Image Is Attached showing it.
![Linter Error While Keeping Link In Same Line](https://github.com/keras-team/keras-cv/assets/84491997/258796d7-e608-4483-92d7-64b1bcbeedd2)

#But The Issue Can be resolved if we keep the Bitly link
I am generating another commit using the Bitly Link. While using Bitly, The Linter tool does not gives an error.
![Using Bitly Link](https://github.com/keras-team/keras-cv/assets/84491997/7c26d773-dfe3-490e-9f38-ca4ad9d5c16c)